addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/Messages.properties

```diff
@@ -327,7 +327,7 @@ automation.error.env.loopvar = Variable {0} has self reference
 automation.error.env.missing = Missing environment: {0}
 automation.error.env.nocontexts = Missing contexts in environment: {0}
 automation.error.env.novar = Variable {0} used but not specified
-automation.error.env.sessionmgmt.engine.bad = Invalid session management scriptEngine: {0}
+automation.error.env.script.engine.bad = Invalid scriptEngine: {0} for: {1}
```

Comment:
I'd suggest use of English "script engine", as a generic method we should not assume how the field is called (e.g. script job could use this method, and it's calling it `engine`).

Reply:
Done